### PR TITLE
Fix installer logging

### PR DESCRIPTION
**Pull request by fm75**
- [ ] Add / update documentation
 - [ ] Add tests

### CODE REVIEW

**Comment by fm75** [comment]
Fixes #241

File: tljh/installer.py [hunk 1]
```diff
@@ -399,7 +399,7 @@ def ensure_config_yaml(plugin_manager):
 
 def main():
     from .log import init_logging
-    init_logging()
+    logger = init_logging()
```
[the remark this answers is not in the document]
Comment (fm75) [reply tljh/installer.py]: I was convinced that the only logging coming out of installer was stderr from the other functions. **not** from logging in them. I commented out what all the functions did. Without that change, "Done!" does not appear on `stderr` or in `installer.log`. 

Yes. logger is global in line 38. I suspect that `bootstrap.py` got separated out and `log.py` was added for the `installer.py` at some point in the early history, but I did not try to dig into that.

The following two lines were posted to `installer.log` in my testing:
```bash
2018-12-03 06:35:20,014 Setting up JupyterHub...
2018-12-03 06:35:20,014 Done!
```
Those lines did not end up in the log without the two changes.

**Comment by fm75** [comment]
I think a better way to fix this is to change `__name__` in the `logging.getLogger(__name__)`
lines in `tljh/installer.py` and `tljh/migrator.py` to `logging.getLogger("tljh")`, to match the logger in the `init_logging` function in `tljh/log.py`

**Comment by yuvipanda** [comment]
@fm75 that feels like a much better idea! Go for it :)

**Pull request closed without merging**